index.html

```diff
@@ -762,18 +845,145 @@ <h2><dfn>CaptureController</dfn></h2>
             [Exposed=Window, SecureContext]
             interface CaptureController {
               constructor();
-              // TODO: Add setFocusBehavior() in a separate PR.
+              undefined setFocusBehavior(CaptureStartFocusBehavior focusBehavior);
             };
           </pre>
           <dl data-link-for="CaptureController" data-dfn-for="CaptureController" class="methods">
             <dt>
               <dfn>constructor</dfn>
             </dt>
+            <dd>
```

Comment:
Instead of constructor steps, you may want to use a table that describes all internal slots as in https://w3c.github.io/web-nfc/#the-ndefreader-object

![image](https://user-images.githubusercontent.com/634478/194590280-7b9655ae-4a29-4512-917a-111211f783b0.png)

Reply:
Looks good. Done.